pinot-core/src/main/java/org/apache/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -198,12 +198,13 @@ public void stop() {
     long schedulerWaitMs = timerContext.getPhaseDurationMs(ServerQueryPhase.SCHEDULER_WAIT);
 
     if (queryLogRateLimiter.tryAcquire() || forceLog(schedulerWaitMs, numDocsScanned)) {
-      LOGGER.info(
-          "Processed requestId={},table={},segments(queried/processed/matched/consuming)={}/{}/{}/{},"
-              + "schedulerWaitMs={},totalExecMs={},totalTimeMs={},minConsumingFreshnessMs={},broker={},"
-              + "numDocsScanned={},scanInFilter={},scanPostFilter={},sched={}",
-          requestId, tableNameWithType, numSegmentsQueried, numSegmentsProcessed, numSegmentsMatched,
-          numSegmentsConsuming, schedulerWaitMs, timerContext.getPhaseDurationMs(ServerQueryPhase.QUERY_PROCESSING),
+      LOGGER.info("Processed requestId:{},table:{},segments(queried/processed/matched/consuming)={}/{}/{}/{},"
```

Comment:
```suggestion
      LOGGER.info("Processed requestId:{},table:{},segments(queried/processed/matched/consuming):{}/{}/{}/{},"
```

Reply:
"=" is better for parsing out since ":" is also present in timestamps. Made this consistent with log in BaseBrokerRequestHandler